app/static/js/courseManagement.js

```diff
@@ -3,13 +3,25 @@ function changeTerm() {
     $('form').submit();
 };
 
+function courseAction(action){
```

Comment:
In general functions should be verbs so you know what they are doing. Something like `executeCourseAction` would be better.